crates/holochain_zome_types/src/fixt.rs

```diff
@@ -376,7 +376,7 @@ fixturator!(
                 let number_of_assigned = rng.gen_range(0..5);
 
                 CapAccess::from((
-                    CapSecretFixturator::new_indexed(Unpredictable, get_fixt_index!()).next().unwrap(),
+                    Some(CapSecretFixturator::new_indexed(Unpredictable, get_fixt_index!()).next().unwrap()),
```

Comment:
Likewise, wondering if `None` would also be a valid point in the unpredictable curve.

Reply:
It would, but it's onerous to make an fixturator for an Option of something, so I opted to just replicate the previous behavior.

Reply:
got it. Even knowing nothing about how fixt works, I kinda anticipated that it would be complicated.